guides/common/modules/proc_creating-discovery-rules.adoc

```diff
@@ -10,7 +10,7 @@ To use the CLI instead of the {ProjectWebUI}, see the xref:cli-creating-discover
 
 .NIC considerations
 Auto provisioning does not currently allow configuring NICs; all systems are being provisioned with the NIC configuration that was detected during discovery.
-However, you can set the NIC in the `{provision-script}` scriplet, using a script, or using configuration management at a later stage.
+However, you can set the NIC in the `{provision-script}` scriptlet, by using a script, or by using configuration management at a later stage.
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Configure* > *Discovery rules*, and select *Create Rule*.
```

Comment:
I know this is not part of your changes. Just wanted to say that 11 steps is a really long procedure!

I recommend turning the steps for the fields (Hosts limit, priority, etc.) into a bullet list. You will save a lot of space and it will be much easier for the user to scan the content.

Reply:
I'll note this for the next round of improvements. Thank you!

Reply:
Included in issue: https://github.com/theforeman/foreman-documentation/issues/3041